test/verify/check-storage-nfs

```diff
@@ -58,8 +60,9 @@ class TestStorageNfs(StorageCase):
         b.wait_text_not("#nfs-mounts tr:contains(/mnt/test) .pf-c-progress__status", "")
 
         # Should be saved to fstab
-        self.assertEqual(m.execute("grep -w nfs /etc/fstab").strip(),
-                         "127.0.0.1:/home/foo /mnt/test nfs defaults")
+        self.maxDiff = None
```

Comment:
is `maxDiff` used anywhere?

Reply:
This line should be removed, I think.

Reply:
I kept it in as otherwise you don't get a diff like in the comment above, but just a message that gives you the first few characters and a "if you want to see the full diff, set maxDiff".

I don't feel strongly about it at all, though -- I'm happy to take it out, and we can put it back in locally if we need to debug it.

Reply:
Ich hab es outgetaked. :grin:

Reply:
Ahh, it's a `unittest` thing? I didn't know that, I thought it's some debugging leftovers.

So, up to you, really. But if we keep it in, it needs a comment.

(Inheritance is wonderful... :-)

Reply:
> So, up to you, really. But if we keep it in, it needs a comment.

Per IRC, let's not.